website/docs/en/blog/index.mdx

```diff
@@ -6,43 +6,49 @@ title: Overview
 
 Check here for the latest articles and release announcements about Rspack.
 
-## [Announcing Rspack v1.0 Alpha](/blog/announcing-1-0-alpha)
+## [Announcing Rspack 1.0](/blog/announcing-1-0)
+
+> August 28, 2024
+
+Today Rspack has finally reached a new milestone - 1.0. This means that Rspack is production ready, covers most of Webpack's APIs and features, and is now prepared to support more users.
+
+## [Announcing Rspack 1.0 Alpha](/blog/announcing-1-0-alpha)
 
 > June 29, 2024
 
 Rspack 1.0 alpha is now available on npm!
 
 Before releasing Rspack 1.0 stable version, we will test for 1~2 months to improve the API stability and reliability of v1.0 and to verify its impact on downstream projects.
 
-## [Announcing Rspack v0.7](/blog/announcing-0-7)
+## [Announcing Rspack 0.7](/blog/announcing-0-7)
```

Comment:
add back the `v`s please.

Reply:
I checked Next.js and Nuxt, they do not use the `s`, so I removed it.

- https://nextjs.org/blog
- https://nuxt.com/blog

Reply:
Just checked React docs as well. appears I'm too old school, Angular still does it, but the rest of the community has moved on.  Fair enough to skip this.